addon/components/rdf-input-fields/heading.hbs

```diff
@@ -0,0 +1,3 @@
+<AuHeading @skin={{this.titleSkin}}>
```

Comment:
I would also pass the `@level` argument so it can be adjusted. Like here: https://github.com/lblod/ember-submission-form-fields/blob/e36be75f1479c5c4a8c792cf69250752fed6b601/addon/components/property-group.hbs#L6